shared/js/media/media_utils.js

```diff
@@ -31,14 +31,14 @@ var MediaUtils = {
     }
     var sizeDecimal = i < 2 ? Math.round(size) : Math.round(size * 10) / 10;
 
-    return navigator.mozL10n.formatValue('byteUnit-' + units[i]).then(
+    return document.l10n.formatValue('byteUnit-' + units[i]).then(
```

Comment:
This shared file is also used by apps/fl/download.html.  So unless you want to add that app to this patch, I think you need something like:
```
(document.l10n || navigator.mozL10n).formatValue(...)
```
That might also be a good idea for the shared dialogs.js file above, even though that is currently only used by gallery and video.